### DISCUSSION TITLE
Movable/Dynamic Chairs

### DISCUSSION THREAD

**Discussion by TheMysteriousGray** [CreativeMD/LittleTiles#1021] [Ideas]
I have no idea how feasible this is, but what I would essentially like is to attach chairs to doors and have the player be able to sit in and move with them. Currently, chairs can be added to door blueprints, but while the model will move with the door, the player remains stuck in place and the door/chair hybrid moves without them. The chair is not usable when the door is open, either, as if the chair's coordinates haven't changed at all.

I feel this feature would be useful for creating things like rotating cranes, moving trains, and other simple rides without players having to stand up and be pushed about by the ride vehicle because they have no place to sit.

Any feedback is appreciated!

-TMG

**Comment by CreativeMD** [comment]
Actually this was a feature, there was just a small issue that prevented it from working. Will be fixed in the newest version. Will try to implement the same for beds.

**Comment by TheMysteriousGray** [comment]
Ah interesting!

ngl, I assumed this was just an oversight or something that hadn't been taken to account, because the basic premise of gluing a chair to a door sounds completely whackadoodle in concept. Glad to know this will be fixed, though!

**Reply by CreativeMD** [reply]
Fix is already out.
